src/Stratis.Bitcoin.Features.Wallet/Controllers/WalletController.cs

```diff
@@ -375,7 +375,8 @@ public IActionResult GetHistory([FromQuery] WalletHistoryRequest request)
                 {
                     var transactionItems = new List<TransactionItemModel>();
 
-                    List<FlatHistory> items = accountHistory.History.OrderByDescending(o => o.Transaction.CreationTime).Take(200).ToList();
+                    List<FlatHistory> items = accountHistory.History.OrderByDescending(o => o.Transaction.CreationTime).ToList();
```

Comment:
@bokobza a bit more readable:

```
IEnumerable<FlatHistory> items = accountHistory.History.OrderByDescending(o => o.Transaction.CreationTime);
if (string.IsNullOrEmpty(request.SearchQuery))
    items = items.Take(200);
```

Also, no reason to `ToList()` unless the intention is to take a copy.

Reply:
```
items = string.IsNullOrEmpty(request.SearchQuery) ? items.Take(200).ToList() : items;`
```

 is as readable as 

```
if (string.IsNullOrEmpty(request.SearchQuery))
    items = items.Take(200);
```

I need ToList, the type I get from `OrderByDescending` is `IOrderedEnumerable<FlatHistory>`.